tutormfe/templates/mfe/build/mfe/Dockerfile

```diff
@@ -33,6 +33,10 @@ RUN echo "copying i18n data" \
 ####################### {{ app_name }} MFE
 ######## {{ app_name }} (git)
 FROM base AS {{ app_name }}-git
+{#- Invalidate the build cache if a change is detected upstream #}
+{%- if app.get("refs") %}
+ADD {{ app["refs"] }}/{{ app.get("version", MFE_COMMON_VERSION) }} /.gitref-{{ app_name }}
```

Comment:
Minor: can we place this file not at the root, maybe in /tmp?

Reply:
Fair enough!